src/workerd/api/modules.h

```diff
@@ -65,11 +65,6 @@ void registerBuiltinModules(jsg::modules::ModuleRegistry::Builder& builder, auto
     builder.add(getExternalUnsafeModuleBundle<TypeWrapper>(featureFlags));
   }
 
-  if (featureFlags.getPythonWorkers()) {
```

Comment:
Moved this into the workerd-api setup for the bundle. Will further refine things in another PR.